packages/@aws-cdk/aws-iot/README.md

```diff
@@ -83,3 +83,16 @@ new iot.TopicRule(this, 'TopicRule', {
 ```
 
 See also [@aws-cdk/aws-iot-actions](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-iot-actions-readme.html) for other actions.
+
+## Thing
+
```

Comment:
can we add a sentence or two on why/when to use a Thing?

Reply:
Yes, we can. I'll write it in this PR!